pkg/polaris-reporting/ctl_polarisreporting.go

```diff
@@ -78,6 +78,13 @@ func (ctl *HelmValuesFromCobraFlags) GetArgs() map[string]interface{} {
 	return ctl.args
 }
 
+// SetArgs set the map to values
+func (ctl *HelmValuesFromCobraFlags) SetArgs(args map[string]interface{}) {
+	for key, value := range args {
+		ctl.args[key] = value
```

Comment:
Do we have to loop over the args here? Can we just set `ctl.args = args` ?

Reply:
for map, you can't do it @mphammer